myokit/_model_api.py

```diff
@@ -4344,7 +4357,8 @@ def lhs(self):
     def promote(self, state_value=0):
         """
         Turns this variable into a state variable with a current state value
-        given by ``state_value``.
+        given by ``state_value``. state_value could be a value or an
+        Expression,
```

Comment:
```suggestion
        given by ``state_value``.
        
        The new ``state_value`` should be either a numerical value or a
        :class:`myokit.Expression`. If an expression is used, it can contain
        references to model variables, as long as they are constant-valued.
```

something like that